features/google/src/model/analyses/ministories/report-activities.js

```diff
@@ -0,0 +1,17 @@
+import { ReportAnalysis } from "@polypoly-eu/poly-analysis";
+import analysisKeys from "../analysisKeys";
+
+export default class ReportActivities extends ReportAnalysis {
+    async analyze({ dataAccount: googleAccount }) {
+        const activities = googleAccount.activityFileInfo;
+        if (activities.length > 0) {
+            googleAccount.reports[analysisKeys.reportActivities] =
```

Comment:
This direct access to implementation is not nice at all. I'll probably do something about it in #977